test/unit/L1/l1Migrator.test.ts

```diff
@@ -920,6 +914,30 @@ describe('L1Migrator', function() {
         await l1Migrator.connect(governor).unpause();
       });
 
+      it('fails to send tx to L2 when caller not governor', async () => {
+        const amount = 200;
+        const seqNo = 7;
+
+        bridgeMinterMock.withdrawETHToL1Migrator.returns(amount);
```

Comment:
I think we can drop these two lines since `migrateLPT()` will never call these functions since a modifier will trigger a revert first.

Reply:
addressed in d865442